packages/orderbook/test/js_api/gui.test.ts

```
@@ -576,31 +576,30 @@ describe('Rain Orderbook JS API Package Bindgen Tests - Gui', async function ()
+			expect(result.error.msg).toBe('Deposit token not found in gui config: token3');
 		});
 
 		it('should remove deposit', async () => {
 			gui.saveDeposit('token1', '50.6');
-			let deposits: TokenDeposit[] = gui.getDeposits();
+			let deposits = extractWasmEncodedData<TokenDeposit[]>(gui.getDeposits());
 			assert.equal(deposits.length, 1);
 
 			gui.removeDeposit('token1');
-			let depositsAfterRemove: TokenDeposit[] = gui.getDeposits();
+			let depositsAfterRemove = extractWasmEncodedData<TokenDeposit[]>(gui.getDeposits());
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'depositsAfterRemove' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)